client/signup/main.jsx

```diff
@@ -291,12 +286,11 @@ class Signup extends React.Component {
 	};
 
 	handleFlowComplete = ( dependencies, destination ) => {
-		debug( 'The flow is completed. Logging you in...' );
+		debug( `The flow is completed. Destination: ${ destination }` );
```

Comment:
super minor, but using a string placeholder avoids the string concat when debug is disabled. like
`debug( 'The flow is completed. Destination: %s', destination )`